tasks/Logger.cpp

```diff
@@ -52,6 +52,7 @@ Logger::Logger(std::string const& name, TaskCore::TaskState initial_state)
     , m_io(0)
 
 {
+    _flush_period.set(base::Time::fromSeconds(10));
```

Comment:
Instead of overwriting the property value here, can't you set 10s as default value when defining the property?

Reply:
No, orogen does not support setting defaults for complex types. Doing it in the constructor is the way to go ...